src/middlewared/middlewared/alert/source/smartd.py

```diff
@@ -15,9 +15,9 @@ def check_sync(self):
             if self.middleware.call_sync("system.vm"):
                 return
 
-            if not self.middleware.call_sync("system.is_freenas"):
-                if self.middleware.call_sync("failover.status") != "MASTER":
-                    return
+            is_ha = self.middleware.call_sync('failover.licensed')
+            if is_ha and not self.middleware.call_sync('failover.status') == 'MASTER':
```

Comment:
`not self.middleware.call_sync('failover.status') == 'MASTER':` seems weird

Also this file uses `"` so let's keep that consistent